packages/calypso-e2e/src/lib/flows/gutenboarding-flow.ts

```diff
@@ -0,0 +1,201 @@
+		const selector = selectors.domainSuggestionSpan( target );
+		const elementHandle = await this.page.waitForSelector( selector );
+
+		// The blog URL is the full inner text of the matching element.
+		const url = await elementHandle.innerText();
+		await elementHandle.click();
+		// Return the selected domain to the caller for further processing.
+		return url;
+	}
+
+	/* Design screen */
+
+	/**
+	 * Given a name, select a design matching the name.
+	 *
+	 * @param {string} name Name of the design to use.
+	 */
+	async selectDesign( name: string ): Promise< void > {
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( selectors.designButton( name ) ),
+		] );
+	}
+
+	/**
+	 * Given one name of the font in a pair, select a matching font peairing.
+	 *
+	 * Example:
+	 * 	- name = Playfair will match [ Playfair / Fira Sans ]
+	 * 	- name = Roboto will match [ Space Mono / Roboto ]
+	 *
+	 * @param {string} name One portion of the name of the font pair.
+	 */
+	async selectFont( name: string ): Promise< void > {
+		// Font selector depends on the viewport name but lumps non-mobile into desktop.
+		const viewportName = getViewportName() === 'mobile' ? 'mobile' : 'desktop';
+
+		// Mobile viewport puts the buttons behind a dropdown.
+		if ( viewportName === 'mobile' ) {
+			await this.page.click( selectors.mobileFontPairingDropdown );
+		}
+
+		// Click on the desired font pairing.
+		await this.page.click( selectors.fontPairingButton( viewportName, name ) );
+
+		if ( viewportName === 'desktop' ) {
+			await this.page.waitForSelector( `button.is-selected span:text("${ name }")` );
+		} else {
+			await this.page.waitForSelector(
+				`button.is-selected-dropdown-option span:text("${ name }")`,
+				{ state: 'hidden' }
+			);
+		}
+	}
+
+	/* Feature selection screen */
+
+	/**
+	 * Given an array of feature strings, selects matching features.
+	 *
+	 * @param {Features} features Array of features to select.
+	 */
+	async selectFeatures( features: Features[] ): Promise< void > {
+		for await ( const feature of features ) {
+			// Click on the feature to select, then make sure it is in fact selected by checking
+			// whether the attribute `is-selected` is present for the button with that feature.
+			await this.page.click( `${ selectors.featureItem } :text-is("${ feature }")` );
+			await this.page.waitForSelector(
+				`${ selectors.featureItem }.is-selected :text-is("${ feature }")`
+			);
+		}
+	}
+
+	/* Plan selection screen */
+
+	/**
+	 * Given a name, select a plan matching the name.
+	 *
+	 * @param {string} name Name of the plan.
+	 */
+	async selectPlan( name: Plans ): Promise< void > {
+		// First, expand the accordion.
+		await this.page.click( ':text-is("Show all plans")' );
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click(
+				`.plans-accordion-item:has(.plans-accordion-item__name:has-text("${ name }")) button:has-text("Select")`
```

Comment:
Small nit (and my fault!): but this should probably go in the top constant selectors as it is not an iteration on another selector

Reply:
Fixed in 80deae5a8cf3c3e1b5f9cb602870245e968d8ab2